### PR TITLE
Royal Guard Knighthood (Ser/Dame)

### PR DESCRIPTION
**Pull request by BadAtThisGame302**
## About The Pull Request

Troll placeholder remains.

Gives the Royal Guard the Ser/Dame (because I'm a beast and was the only guy arguing on the Captain PR for it so I did a when you code it moment) honorary like the Captain.

Gives them a better and more luxurious description from a single sentance.

And changes their path from manorguard to royalguard.

## Why It's Good For The Game

Turns the Royal Guard into minature Knights which is based, plus the Captain PR was testmerged for quite afew rounds so people just continue to call Royal Guards with Ser infront of their names. Also gives them a more expanded description to accomodate.

Troll bugfix.

## Pre-Merge Checklist

- [ ] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

### CODE REVIEW

**Review by Schmorgisborg — APPROVED** [review]
Looks good to me, no compilation errors, and no sneaky additions.
Creative and concise- something we could all do with a little more of.

File: code/modules/jobs/job_types/roguetown/garrison/manorguard.dm [hunk 1]
```diff
@@ -15,31 +15,38 @@
...
 			var/index = findtext(H.real_name, " ")
 			if(index)
 				index = copytext(H.real_name, 1,index)
 			if(!index)
 				index = H.real_name
-			S.name = "royal guard's tabard ([index])"
+			S.name = "knight's tabard ([index])"
+		var/prev_real_name = H.real_name
+		var/prev_name = H.name
+		var/honorary = "Ser"
```
Comment (ThePainkiller) [inline code/modules/jobs/job_types/roguetown/garrison/manorguard.dm:42]: Still no idea why — if we're introducing a female title for knight — does the male version have to become a made up gender neutral term. Why not "Sir", as it should befit medieval fantasy, for knights?

File: code/modules/jobs/job_types/roguetown/garrison/manorguard.dm [hunk 2]
```diff
@@ -15,31 +15,38 @@
 		"Aasimar"
 	)
 	allowed_ages = list(AGE_ADULT, AGE_MIDDLEAGED)
-	tutorial = "An elite guardsman with extra training. Protect the nobles."
+	tutorial = "You're an elite member of the Garrison, awarded with a knighthood for your prowess and loyalty. Be a stalwart guardian for the royal family, and serve them until your dying breath."
```
Comment (ThePainkiller) [inline code/modules/jobs/job_types/roguetown/garrison/manorguard.dm:18]:
```suggestion
	tutorial = "You're an elite member of the Garrison, awarded with knighthood for your prowess and loyalty. Be a stalwart guardian for the royal family, and serve them until your dying breath."
```

**Review by ThePainkiller — CHANGES_REQUESTED** [review]

File: code/modules/jobs/job_types/roguetown/garrison/manorguard.dm (hunk 1 above)
Comment (BadAtThisGame302) [reply code/modules/jobs/job_types/roguetown/garrison/manorguard.dm:42]: I think the title 'Ser' comes from Game of Thrones, which there they are called Sers not Sirs (the knights), so it's a nudge to that medieval sort of fantasy. 

I personally prefer Ser too, it makes it more... homely? It feels like what an uneducated peasant would call an actual Sir Knight.

Comment (ThePainkiller) [reply code/modules/jobs/job_types/roguetown/garrison/manorguard.dm:42]: Dear god Game of Thrones? I'm gonna be biased on saying this but no wonder it sounded horrible to me, and now I understand **why**.
Peasants, and people on medieval settings would normally use **_"sire"_** instead, which is a literal archaic version of "sir", and more period appropiate. "Ser" sounds like the elder Oblivion beggars muttering on their own version of the english language a misspronunciation of "sir". Yuck.

File: code/modules/mob/living/simple_animal/rogue/creacher/troll.dm [hunk 3]
```diff
@@ -45,7 +45,6 @@
 //	stat_attack = UNCONSCIOUS
 	remains_type = /obj/item/rogueweapon/battle
```
Comment (ThePainkiller) [inline code/modules/mob/living/simple_animal/rogue/creacher/troll.dm:46]:
```suggestion
	remains_type = /obj/effect/decal/remains/human // Placeholder until Troll remains are sprited.
```

**Review by ThePainkiller — CHANGES_REQUESTED** [review]
Since you're, for some reason, also touching `troll.dm`

**Pull request merged**